### DISCUSSION TITLE
(4일차) DispatchWorkItem DispatchGroup

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#150] [General]
DispatchQueue 세분화하고 task를 세분화하고 세분화하여 만들 수 있는 DispatchWorkItem
그리고 DispatchGroup는 에니메이션이 끝나고 모달을 올리는 작업을 할 때 유용할 수 있습니다

때문에  DispatchWorkItem DispatchGroup을 잘 정리하고 한번 개념을 잘 잡고 가시기를 바랍니다! 🦔

**Comment by Eunho0922** [comment]
어제 몸 상태가 너무 안좋아서 못올렸습니다.. 오늘 과제하고 같이 올리겠습니다
